orchagent/fdborch.cpp

```diff
@@ -36,6 +37,73 @@ FdbOrch::FdbOrch(DBConnector *db, string tableName, PortsOrch *port) :
     Orch::addExecutor(fdbNotifier);
 }
 
+bool FdbOrch::bake()
+{
+    Orch::bake();
+
+    auto consumer = dynamic_cast<Consumer *>(getExecutor(APP_FDB_TABLE_NAME));
+    if (consumer == NULL)
+    {
+        SWSS_LOG_ERROR("No consumer %s in Orch", APP_FDB_TABLE_NAME);
+        return false;
+    }
+
+    size_t refilled = consumer->refillToSync(&m_fdbStateTable);
+    SWSS_LOG_NOTICE("Add warm input FDB State: %s, %zd", APP_FDB_TABLE_NAME, refilled);
+    return true;
```

Comment:
does this call fdb notification? here the consumer is the app db? how can the fdb entries in state db got translated into fdb notificastions?

Reply:
1. Are you asking syncd receive this fdb entry and send orchagent notification? No.
2. Yes, the consumer is the app db
3. Never translated.

---
In reply to: [219421390](https://github.com/Azure/sonic-swss/pull/615#discussion_r219421390) [](ancestors = 219421390)